cdap-ui/app/cdap/components/Management/index.js

```diff
@@ -33,7 +33,7 @@ var shortid = require('shortid');
 var classNames = require('classnames');
 
 var dummyData = {
-  version: '4.0-SNAPSHOT',
+  version: '4.0.0',
```

Comment:
I guess its unrelated to this PR, and there's more stuff to integrate on this page, so we could do this in a separate PR.